packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -128,7 +131,22 @@ application's performance is enhanced as the operation is performed only once, t
 
 _Methods_
 
--   none
+-   `hideFooter()` - hides the footer section that contains all formatting options and the `footer-actions` slot.
+-   `clearContent()` - clears the content in the editor through [clearContent()](https://tiptap.dev/api/commands/clear-content) command in Tiptap.
+
+_Open Discussion_
+
+-   We are proposing a design that involves displaying only placeholder text initially and footer section will be hidden. Upon focusing the editor, we will capture
+    Tiptap's [focus event](https://tiptap.dev/api/events#focus) to show the footer section. As per the
+    UX for comments feature, we will revert back to the original state (i.e,. without the footer section) only if the user clicks the `cancel` or `ok` button. Therefore,
+    we came to this decision of exposing the above `hideFooter()` and `clearContent()` methods.
+-   There isn't a specific reason to expose two separate methods when considering the comments feature; they could be combined into a single method. However, we are uncertain about the
```

Comment:
Reset is a comment use-case specific operation. If we were using this as a description text editor it would still be reasonable to cancel the edit to reset to the previous contents and hide the footer until focus, but it would only be a clear if the previous contents were empty. These sound like decisions for the application, not the rich text editor. So I agree with the decision to keep these methods separate.

Reply:
Closing this comment as the `clearContent()` method is removed as we can achieve it by setting the `markdown` value to an empty string.